src/main/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExceptionMapper.java

```diff
@@ -143,6 +145,16 @@
         return errorDetailsResponseEntity(ex, FORBIDDEN, ACCESS_EXCEPTION.getErrorMessage());
     }
 
+    @ExceptionHandler(RequestRejectedException.class)
```

Comment:
Are these exceptions used anywhere ?

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/444/commits/5266fb5901332c8363777707d56f6e1941266ac5